control_panel_api/management/commands/migrate_lambda_data_0_dump_user_policies.py

```diff
@@ -0,0 +1,58 @@
+import json
+import logging
+
+import boto3
+from botocore.exceptions import ClientError
+from django.core.management.base import BaseCommand
+
+from control_panel_api.models import User
+
+
+READWRITE = 'readwrite'
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    NOTE: Needs permission to perform `iam:ListAttachedRolePolicies` action
+    """
+
+
+    help = ("For each of the users in the DB, it collects its attached IAM "
+            "policies and prints this data to stdout")
```

Comment:
Just an idea - it might be good to have `help` be equal to `__doc__` and put the help text in the class doc comment. Eg:
```python
class Command(BaseCommand):
    """
    For each of the users in the DB, ...
    NOTE: Needs permission ...
    """
    help = __doc__

Reply:
👍